e2e-test/log/build.gradle

```diff
@@ -159,4 +160,10 @@ dependencies {
     integrationTestImplementation "com.linecorp.armeria:armeria:1.0.0"
     integrationTestImplementation "org.awaitility:awaitility:4.1.1"
     integrationTestImplementation "org.opensearch.client:opensearch-rest-high-level-client:${versionMap.opensearchVersion}"
-}
\ No newline at end of file
+}
+
+configurations.all {
```

Comment:
Is there a reason we are this forcing resolution here instead of in the root project?

Reply:
It's only a dependency on only `:e2e-test:log`. I thought it best to scope the forced version. If preferable I can force it for the entire project.

Reply:
OK. If only this test needs `snakeyaml`, then it should be here. I do still wonder if we can just exclude the Android dependency instead though.